Python/Product/Debugger/CustomDebugAdapterProtocolExtension.cs

```diff
@@ -87,6 +89,23 @@ public static void SendRequestSync<TArgs>(DebugRequest<TArgs> request)
         public void RegisterCustomMessages(ICustomMessageRegistry registry, IProtocolHostOperations hostOperations) {
             _hostOperations = hostOperations;
 
+            registry.RegisterEventType<VersionInfoEventArgs>(OnVersionInfo);
+            registry.SetInitializeRequestProperty("supportsVersionInfo", true);
+        }
+
+        private void OnVersionInfo(VersionInfoEventArgs e) {
+            if (PackageVersion.TryParse(e.Version, out PackageVersion runningVersion)) {
+                var bundledPtvsdVersionStr = "4.0.0a4";
+                var bundledPtvsdVersion = PackageVersion.Parse(bundledPtvsdVersionStr);
+                if(runningVersion.CompareTo(bundledPtvsdVersion) < 0) {
+                    MessageBox.Show(
+                        new VSWin32Window(Process.GetCurrentProcess().MainWindowHandle),
+                        Strings.InstalledPtvsdOutdatedMessage.FormatInvariant(e.Version, bundledPtvsdVersionStr),
```

Comment:
We typically use `Strings.ProductTitle` for the caption.
I think it should be FormatUI instead of FormatInvariant.